cmd/main.go

```diff
@@ -21,7 +20,7 @@ func RunCLI() error {
 	app := cli.NewApp()
 	app.Name = "summon"
 	app.Usage = "Parse secrets.yml and export environment variables"
-	app.Version = summon.VERSION
+	app.Version = command.AllVersions
```

Comment:
You shouldn't co-opt this field. Summon's version is not related to available provider versions.

Reply:
https://github.com/cyberark/summon/issues/56